.github/workflows/continuous_integration.yml

```diff
@@ -58,11 +58,11 @@ jobs:
         pip install git+https://github.com/goodchemistryco/Tangelo.git@develop
       if: always()
 
-    - name: Download required data to run the MIFNO notebook
+    - name: Download required data to run the iFCI - MIFNO notebook
       run: |
         cd examples/problem_decomposition
         sudo apt-get install subversion
-        svn export https://github.com/goodchemistryco/Tangelo/branches/develop/tangelo/problem_decomposition/tests/incremental/data
+        svn export https://github.com/goodchemistryco/Tangelo/branches/develop/tangelo/problem_decomposition/tests/incremental/data/BeH2_CCPVDZ_MIFNO_HBCI
```

Comment:
note: you never use the IFCI folder.

Reply:
We use it in the Tangelo unittests, but not in this notebook.